train.py

```diff
@@ -0,0 +1,99 @@
+import os
+import sys
+import time
+import argparse
+
+import torch
+import psutil
+import imageio
+import numpy as np
+from torchvision.utils import save_image
+
+from utils import get_interpolations
+from models.autoencoder import Autoencoder
+from train_utils import test_epoch, train_epoch
+
+parser = argparse.ArgumentParser(description="Main function to call training for different AutoEncoders")
```

Comment:
Please check out [hydra](https://hydra.cc/docs/intro/) - I'm trying to use it to control various experiments and so far it's really cool. You can check it out in other projects we're working on right now:

https://github.com/Nospoko/ecg-segmentation-unet/tree/MIDI-24/training
https://github.com/Nospoko/bert-midi-evals/tree/MIDI-18/bag-of-pitches